torch/distributed/distributed_c10d.py

```diff
@@ -1759,15 +1765,25 @@ def broadcast_object_list(object_list, src=0, group=None):
     else:
         object_sizes_tensor = torch.empty(len(object_list), dtype=torch.long)
 
+    # Current device selection.
+    # To preserve backwards compatibility, dist_device is default to None in
```

Comment:
nit: might also want to add \`\` \`\` around `dist_device`, `None`, and `current_device` in this paragraph.

Reply:
SG